pkg/asset/cluster/cluster.go

```diff
@@ -59,9 +59,8 @@ func (c *Cluster) Dependencies() []asset.Asset {
 func (c *Cluster) Generate(parents asset.Parents) (err error) {
 	installConfig := &installconfig.InstallConfig{}
 	terraformVariables := &TerraformVariables{}
-	adminKubeconfig := &kubeconfig.Admin{}
```

Comment:
If we are not using the kubeconfig admin, can we drop it from the dependencies?

also regenerate the graph at https://github.com/openshift/installer/blob/master/docs/design/assetgeneration.md#dependency-graph